test/test_types_helper.hpp

```diff
@@ -84,7 +84,7 @@ inline ObjectId TestValueGenerator::convert_for_test<ObjectId>(int64_t v)
     uint64_t cur = static_cast<uint64_t>(v);
     for (size_t i = 0; i < 24; ++i) {
         value += char(hex_digits[cur % 16]);
-        cur -= (cur % 16);
+        cur = cur >> 1;
```

Comment:
This test code is used in the benchmarks and was generating many of the same values because as soon as a value is a multiple of 16 we'd subtract 0 which left all remaining values filled with 0s. This skewed results. It might affect a few of the other ObjectId benchmarks a bit compared to our historic results, but is much more realistic.